src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationHandlers.Userinfo.cs

```diff
@@ -519,6 +519,21 @@ public ValueTask HandleAsync(ExtractUserInfoResponseContext context)
                         context.Response[Claims.GivenName] = name["first"];
                     }
                 }
+                
```

Comment:
In general, we only use this handler and fix up non-standard claims here for providers that implement a "standard OIDC userinfo" (or claim to 😄) but that have compliance issues (e.g the node type of a standard claim isn't correct). Otherwise, it's simpler/better not to amend the returned claims here and only do the non-standard -> WS-Federation claims mapping in the `MapCustomWebServicesFederationClaims` handler.